lib/tapioca/rbi/rewriters/sort_nodes.rb

```diff
@@ -11,11 +11,11 @@ class SortNodes < Visitor
         def visit(node)
           return unless node.is_a?(Tree)
           visit_all(node.nodes)
+          original_order = original_nodes_order(node.nodes.dup)
```

Comment:
Passing a copy so we always preserve the original order since the array will be sorted in place.